### PR TITLE
Mixed

### PR DESCRIPTION
**Pull request by nouiz**
a few mixed stuff

### CODE REVIEW

File: doc/tutorial/examples.txt [hunk 1]
```diff
@@ -465,11 +465,19 @@ There are :ref:`other distributions implemented <libdoc_tensor_raw_random>`.
...
+ The RandomStream only work on the CPU, MRG31k3p
 work on the CPU and GPU. CURAND only work on the GPU.
 
+.. note::
+
+    To use you the MRG version easily, you can just change the import to:
+
+        .. code-block:: python
+
+            from theano.sandbox.rng_mrg import MRG_RandomStreams
```
Comment (lamblin) [inline doc/tutorial/examples.txt]: Why not `... as RandomStreams`?

File: theano/sandbox/cuda/opt_util.py [hunk 2]
```diff
@@ -73,8 +73,21 @@ def opt(node):
                 if lr is None or targ is None:
                     return None
                 inputs = list(targ.inputs)
-                inputs[alpha_in] = lr * targ.inputs[alpha_in]
-                inputs[beta_in] = lr * targ.inputs[beta_in]
+                try:
+                    raise NotScalarConstantError()
```
Comment (lamblin) [inline theano/sandbox/cuda/opt_util.py]: Is that supposed to be the final version? The code below will always be skipped.

**Comment by nouiz** [comment]
i fixed both comment.

**Comment by lamblin** [comment]
Thanks!
I'm OK to merge when Travis passes.

**Pull request merged**